src/utils/shell.ts

```diff
@@ -4,8 +4,19 @@ import * as os from 'os';
 // @ts-ignore
 import shellPath from 'shell-path';
 
+import {ResourceKind} from '@models/resourcekindhandler';
+
 let cachedShellPath: string | undefined;
 
+// Documentation links hash-table
+export const resourceKindDocLinks: {[name: string]: string} = {
+  ConfigMap: 'https://kubernetes.io/docs/concepts/configuration/configmap/',
```

Comment:
let's add this link to the ResourceKindHandler interface - i.e. a helpLink property which we can set on all corresponding handlers - and then query the handler for the current resource for this link!?

Reply:
Done!